sensor/common/centralclient/client_test.go

```diff
@@ -28,22 +28,10 @@ import (
 const (
 	endpoint = "localhost:8000"
 
-	// 1. Deploy a new StackRox instance
-	// 2. Create and generate a new CA and replace it in the additional-ca.yaml:
-	// $ openssl genrsa -des3 -out myCA.key 2048
-	// $ openssl req -x509 -new -nodes -key myCA.key -sha256 -out myCA.pem -days 100000 -subj '/CN=Root LoadBalancer Certificate Authority'
-	// $ kubectl -n stackrox apply -f additional-ca.yaml
-	//
-	// 3. Receiving trust info examples from a running cluster:
+	// To create new data you can do it manually via roxcurl, or use ./update-testdata.sh
 	// $ roxcurl /v1/tls-challenge?"challengeToken=h83_PGhSqS8OAvplb8asYMfPHy1JhVVMKcajYyKmrIU="
```

Comment:
If this token needs to be in sync with the one in the script, please add comments.

Reply:
done